content/guides/docker-concepts/the-basics/what-is-a-registry.md

```diff
@@ -0,0 +1,135 @@
+---
+title: What is a registry?
+keywords: concepts, build, images, container, docker desktop
+description: What is a registry? This Docker Concept will explain what a registry is, explore their interoperability, and have you interact with registries.
+---
+
+{{< youtube-embed 2WDl10Wv5rs >}}
+
+## Explanation
+
+Now that you know what a container image is and how it works, you might wonder - where do you store these images? 
+
+Well, you can store your container images on your computer system, but what if you want to share them with your friends or use them on another machine? That's where the image registry comes in.
+
+An image registry is a centralized location for storing and sharing your container images. It can be either public or private. [Docker Hub](https://hub.docker.com) is a public registry that anyone can use and is the default registry. 
+
+While Docker Hub is a popular option, there are many other available container registries available today, including [Amazon Elastic Container Registry(ECR)](https://aws.amazon.com/ecr/), [Azure Container Registry (ACR)](https://azure.microsoft.com/en-in/products/container-registry), and [Google Container Registry (GCR)](https://cloud.google.com/artifact-registry). You can even run your private registry on your local system or inside your organization. For example, Harbor, JFrog Artifactory, GitLab Container registry etc.
+
+### Registry vs repository
+
+While you're working with registries, you might hear the terms **registry** and **repository** as if they're interchangeable. Even though they're related, they're not quite the same thing.
+
+A _registry_ is a centralized location that stores and manages container images, whereas a _repository_ is a collection of related container images within a registry. Think of it as a folder where you organize your images based on projects. Each repository contains one or more container images.
+
+>**Note**
+>
+> You can create one private repository and unlimited public repositories using the free version of Docker Hub. For more information, visit the [Docker Hub subscription page](https://www.docker.com/pricing/).
+
+## Try it now
+
+In this hands-on, you will learn how to build and push a Docker image to the Docker Hub repository.
+
+### Sign up for a free Docker account
+
+1. If you have not done so, [create a Docker account](https://hub.docker.com/signup).
+
+    ![Screenshot of the official Docker Hub page showing the Sign up page](images/dockerhub-signup.webp?border)
```

Comment:
Suggest updating the image. For me, the image doesn't match where the link took me.

Reply:
I adjusted the wording to align with the image. Hope it looks good.